innovation.game.php

```diff
@@ -12235,6 +12313,17 @@ function stInterSelectionMove() {
                 }
                     
                 break;
+
+            // id 174, Artifacts age 6: Marcha Real
+            case "174N1A":
+                $card_id = self::getGameStateValue('id_last_selected');
+                if (self::getGameStateValue('auxiliary_value') == -1) {
+                    // Log the card that is revealed first
+                    self::setGameStateValue('auxiliary_value', $card_id);
+                }
+                self::transferCardFromTo(self::getCardInfo($card_id), $player_id, 'revealed');
```

Comment:
I don't think you'll need this

Reply:
if the revealed,deck thing works, yes I think you are right.  However, the minor step interrupt means that the action from the initial $options array isn't performed so you need to do this.

Reply:
I can play with it some more

Reply:
After thinking about this a little more, I think it is optimal the way it is given the inter move interruption that occurs.

If you can think of a more elegant way to track the cards that are used without using that inter move operation then I am all ears. :)